src/styles/StyleUtils.js

```diff
@@ -162,7 +162,7 @@ function getAvatarBorderStyle(size, type) {
  * @returns {Object}
  */
 function getDefaultWorkspaceAvatarColor(workspaceName) {
-    const colorHash = ReportUtils.hashLogin(workspaceName.trim(), workspaceColorOptions.length);
+    const colorHash = UserUtils.hashLogin(workspaceName.trim(), workspaceColorOptions.length);
```

Comment:
This really begs the question if we need this to be in a lib at all. It's one line of code that is only used in one spot. Let's just define it here and keep it out of the utils entirely. This is a good reminder of over-engineering things :D (not you, I'm not sure who originally wrote this).

Reply:
Ah, OK. I missed that, thanks.